### ISSUE TITLE
SEO friendly infinite scroll

### ISSUE THREAD

**Issue by kontrollanten** [Chocobozzz/PeerTube#6332]
### Describe the problem to be solved

[Ahrefs](https://ahrefs.com) is reporting orphan pages which means they find pages in sitemap.xml whom isn't linked from somewhere.

![image](https://github.com/Chocobozzz/PeerTube/assets/6680299/76cff11c-9d3a-482e-962c-790591f1171e)

### Describe the solution you would like

Make all video pages crawlable. https://developers.google.com/search/blog/2014/02/infinite-scroll-search-friendly

**Comment by Chocobozzz** [comment]
Hi,

[Ahrefs](https://ahrefs.com/) seems to be an interesting tool. Can we try to fix all these problems? URLs examples would help me

> Make all video pages crawlable

If I understand correctly, we list in the sitemap video URLs that cannot be fetched from regular videos list pages because the crawler should scroll the page?

**Comment by kontrollanten** [comment]
> If I understand correctly, we list in the sitemap video URLs that cannot be fetched from regular videos list pages because the crawler should scroll the page?

Yes, that's how I understand it. I think a solution would be to add a "Load more" link in the bottom of the feed who links to "?page=X". This would also add support to direct link to a specific page in the feed, like this https://scrollsample.appspot.com/items?page=8

**Comment by Chocobozzz** [comment]
Implemented in https://github.com/Chocobozzz/PeerTube/commit/006bbf2b52f845fcabf47de6e83590bda258f936
